src/samplers/inversioncdt.jl

```diff
@@ -0,0 +1,54 @@
+"""
+    A small helper struct representing an identity permutation.
```

Comment:
```suggestion
A small helper struct representing an identity permutation.
```
Indenting this will incorrectly render the prose as a code block